redisinsight/api/src/modules/browser/keys/key-info/strategies/list.key-info.strategy.spec.ts

```diff
@@ -81,5 +95,29 @@ describe('ListKeyInfoStrategy', () => {
 
       expect(result).toEqual({ ...getKeyInfoResponse, size: null });
     });
+
+    it('should not check size when length >= 50,000', async () => {
```

Comment:
Maybe it's worth to have a scenario with 50000+ as well. `it.each` can do the job